scripts/maint/practracker/util.py

```diff
@@ -14,7 +18,7 @@ def get_tor_c_files(tor_topdir, exclude_dirs):
 
             # Exclude the excluded paths
             full_path = os.path.join(root,filename)
-            if any(exclude_dir in full_path for exclude_dir in exclude_dirs):
+            if any(exclude_dir in full_path for exclude_dir in EXCLUDE_SOURCE_DIRS):
```

Comment:
I don't like looking for the excluded path *everywhere* in the full_path string; it should only be excluded when it is right under TOP_SRCDIR.  In other words, if tor is in `~/tor/`, then `~/tor/src/trunnel/` should be excluded, but "~/src/tor/src/lib/handle_trunnel/src/trunnel` shouldn't be excluded

Reply:
I tried to do this, but I think it should be done properly with tests to make sure of it's correctness.
Added to post-merge ticket #29746.